cantabular/queries.go

```diff
@@ -212,32 +212,25 @@ query($dataset: String!, $variables: [String!]!) {
 	}
 }`
 
-// QueryGeographyDimensions is the graphQL query to obtain geography dimensions (subset of variables, without categories)
-const QueryGeographyDimensions = `
-query($dataset: String!, $limit: Int!, $offset: Int) {
+// QueryNonGeoDimensionsByName is the graphQL query to obtain dimensions by name (subset of variables, without categories)
+// but excluding geography dimensions
+const QueryNonGeoDimensionsByName = `
+query($dataset: String!, $variables: [String!]!) {
 	dataset(name: $dataset) {
-		variables(rule: true, skip: $offset, first: $limit) {
-			totalCount
+		variables(names: $variables, rule: false) {
 			edges {
 				node {
 					name
-					description
-					meta{
```

Comment:
thismeta bit deffo needs to go? 
speaking in reference to the latest slack huddle